packages/console-sdk/src/MicrofrontendIntegrator.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  Collections,
+  Endpoints,
+  IEnvironment,
+  IProject,
+  PublicVariable,
+  ServiceConfigMap,
+  Services,
+} from '@mia-platform-internal/console-types'
+import { QiankunProps } from 'vite-plugin-qiankun/dist/helper'
```

Comment:
I'd avoid to import vite in this package, there is a specific package for vite

Reply:
is the import that the creator suggest inside the docs
https://www.npmjs.com/package/vite-plugin-qiankun